### PR TITLE
Added flight requirement to sapphiron quest

### PR DESCRIPTION
**Pull request by vampirika**

### CODE REVIEW

File: src/WarcraftLegacies.Source/Quests/Scourge/QuestSapphiron.cs [hunk 1]
```diff
@@ -33,7 +34,7 @@ public sealed class QuestSapphiron : QuestData
       "Sapphiron has been slain, and has been reanimated as a mighty Frost Wyrm under the command of the Scourge.";
 
     /// <inheritdoc/>
-    protected override string RewardDescription => "Unlock one of the requirements to train Frost Wyrms. " +
+    protected override string RewardDescription => "Unlock ability to train Frost Wyrms. " +
```
Comment (YakaryBovine) [inline src/WarcraftLegacies.Source/Quests/Scourge/QuestSapphiron.cs:37]: While you're in there can you change this to the standard format, i.e. "Learn to train x from the y", and use GetObjectName for x and y?

Comment (vampirika) [reply src/WarcraftLegacies.Source/Quests/Scourge/QuestSapphiron.cs]: Updated

**Comment by YakaryBovine** [comment]
Just on my phone so might be viewing things wrong, but I think you need to revert those war3mapmisc.txt changes. Not sure why those happen, it's happened to me a couple of times.

**Comment by vampirika** [comment]
> Just on my phone so might be viewing things wrong, but I think you need to revert those war3mapmisc.txt changes. Not sure why those happen, it's happened to me a couple of times.

Done, dunno why that happened/happens

**Review by YakaryBovine — APPROVED** [review]
Looks great!

**Pull request merged**